opentelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## vNext
 
+- **BREAKING** [#1993](https://github.com/open-telemetry/opentelemetry-rust/pull/1993) Box complex types in AnyValue enum
+Before:
```

Comment:
It looks like you have mistakenly switched Before and After.

Reply:
Oh good catch. Fixed now :)